### ISSUE TITLE
Pretrained KITTI Model

### ISSUE THREAD

**Issue by samuel-woerz** [gangweiX/ACVNet#8]
Hi, great work and thanks for providing your code.
Do you plan on releasing the weights for the model finetuned on KITTI?
Also, did you freeze the attention weights during KITTI finetuning?

**Comment by gangweix** [comment]
No freeze the attention weights during KITTI finetuning.

**Comment by samuel-woerz** [comment]
Thanks for your answer!

**Comment by FuzzyDunl0p** [comment]
I am currently trying to reproduce your KITTI finetuning and I have three questions:

1. After reducing the learning rate in the 300th epoch of the mixed KITTI2012 and KITTI2015 finetuning, do you reset the learning rate to 0.001 when you start the 500 epoch finetuning or do you continue with the reduced learning rate?
2. After the 300 epochs, do you reduce the learning rate by a factor of 2, as stated in the paper, or by a factor of 10, as GwcNet did it? Reducing by a factor of 10 performs better for me.
3. My KITTI15 validation EPE peaks at 0.574 during the 500 mixed epochs, and again peaks at 0.57 during the additional 500 KITTI15 only epochs. Are these EPEs similar to what you got?

Thank you for your time!

**Comment by samuel-woerz** [comment]
I am currently trying to reproduce your KITTI finetuning and I have three questions:

1. After reducing the learning rate in the 300th epoch of the mixed KITTI2012 and KITTI2015 finetuning, do you reset the learning rate to 0.001 when you start the second 500 epoch finetuning or do you continue with the reduced learning rate?
2. After the 300 epochs, do you reduce the learning rate by a factor of 2, as stated in the paper, or by a factor of 10, as GwcNet did it? Reducing by a factor of 10 performs better for me.
3. My KITTI15 validation EPE peaks at 0.574 during the 500 mixed epochs, and again peaks at 0.57 during the additional 500 KITTI15 only epochs. Are these EPEs similar to what you got?

Thank you for your time!

**Comment by kangarooCV** [comment]
Hello, when will you upload the pre-model of kitti15 dataset? Thanks!

**Comment by UntrainedButLoveCode** [comment]
所以有没有kitti15的权重啊？
